release-content/0.15/release-notes/15800_Add_mesh_picking_backend_and_MeshRayCast_system_parameter.md

```diff
@@ -1,4 +1,117 @@
 <!-- Add mesh picking backend and `MeshRayCast` system parameter -->
 <!-- https://github.com/bevyengine/bevy/pull/15800 -->
 
-<!-- TODO -->
+TODO: Laser beam image
+
+Being able to click on objects to select them is a vital and seemingly simple task in any game.
+Since 2020, doing this in Bevy has largely meant pulling in `@aevyrie`'s beloved ecosystem crate, [`bevy_mod_picking`] and it's simple raycasting companion [`bevy_mod_raycast`].
```

Comment:
```suggestion
Since 2020, doing this in Bevy has largely meant pulling in `@aevyrie`'s beloved ecosystem crate, [`bevy_mod_picking`] and its simple raycasting companion [`bevy_mod_raycast`].
```